### ISSUE TITLE
First header of README.md not very visible in GitHub's dark theme

### ISSUE THREAD

**Issue by projjalm** [dino/dino#1450]
GitHub light:
![Screenshot 2023-06-18 at 23-31-27 dino_dino Modern XMPP ( Jabber ) Chat Client using GTK _Vala](https://github.com/dino/dino/assets/121298637/23148c4c-3a06-4571-896f-40a684991670)

GitHub dark:
![Screenshot 2023-06-18 at 23-30-32 dino_dino Modern XMPP ( Jabber ) Chat Client using GTK _Vala](https://github.com/dino/dino/assets/121298637/b4dae72b-4066-4354-bdf0-d07ba56e702b)  

One fix could be changing the text color to something more moderate or near to the color of Dino's logo (#05988a).

**Comment by licaon-kter** [comment]
That's not text but a SVG `https://dino.im/img/readme_header.svg`

**Comment by projjalm** [comment]
@licaon-kter Right, if it was plain GFM it couldn't be colored in first place... https://github.com/github/markup/issues/1440 :(

**Comment by Commenter25** [comment]
This can actually be solved quite easily! I was going to make a pull request myself, but I realized the images are not stored in this repo unfortunately.

One solution is two separate images, using the `<picture>` tag.
```html
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="image-dark.png">
  <img src="image.png">
</picture>
```

Another more efficient method, since this is an SVG, is to have the SVG itself dynamically change. This could be done by adding a class to the two text groups, like `class="text"`, and then adding a style tag above defs:
```html
<style>
.text { fill: #000 }
@media ( prefers-color-scheme: dark ) {
  .text { fill: #fff }
}
</style>
```

**Comment by projjalm** [comment]
Voting for the second one. :)

**Comment by fiaxh** [comment]
Fixed in b490eb6
